src/ragas/metrics/_factual_correctness.py

```diff
@@ -151,23 +157,77 @@ class DecompositionType(Enum):
 )
 
 
-class ClaimDecompositionPrompt(
-    PydanticPrompt[ClaimDecompositionInput, ClaimDecompositionOutput]
-):
-    instruction = """
-    Decompose and break down each of the input sentences into one or more standalone statements. Each statement should be a standalone claim that can be independently verified.
-    Follow the level of atomicity and coverage as shown in the examples.
-    """
-    input_model = ClaimDecompositionInput
-    output_model = ClaimDecompositionOutput
+# ============================================================================
+# DIRECT PROMPT TEMPLATES (No PydanticPrompt dependencies)
+# ============================================================================
+
+
+def _generate_claim_decomposition_prompt(
+    atomicity: str, coverage: str, response: str
+) -> str:
+    """Generate claim decomposition prompt based on atomicity and coverage levels."""
+
+    # Get examples for the specified atomicity and coverage
+    decomposition_type = DecompositionType(f"{atomicity}_atomicity_{coverage}_coverage")
+    examples = claim_decomposition_examples.get(decomposition_type, [])
+
+    # Build examples section
+    examples_text = ""
+    if examples:
+        examples_text = "\n--------EXAMPLES-----------\n"
+        for i, (input_example, output_example) in enumerate(examples, 1):
+            examples_text += f"Example {i}\n"
+            examples_text += f'Input: {{"response": "{input_example.response}"}}\n'
+            examples_text += (
+                f'Output: {{"claims": {json.dumps(output_example.claims)}}}\n\n'
+            )
+        examples_text += "-----------------------------\n"
+
+    return f"""Decompose and break down each of the input sentences into one or more standalone statements. Each statement should be a standalone claim that can be independently verified.
```

Comment:
Move the prompts to it's own director and use .format to fill in the templates or wrap in a function there